### PR TITLE
Crm mon add fence history 1.1

### PR DESCRIPTION
**Pull request by wenningerk**
#1521

### CODE REVIEW

**Comment by wenningerk** [comment]
synced with #1521

File: xml/crm_mon.rng [hunk 1]
```diff
@@ -200,6 +200,56 @@
...
+                                <attribute name="exec-reason"> <text/> </attribute>
+                            </optional>
+                            <optional>
+                                <!-- details on the execution like failure reason -->
+                                <attribute name="exec-details"> <text/> </attribute>
+                            </optional>
+                        </element>
+                    </zeroOrMore>
+                </element>
+            </optional>
```
Comment (jnpkrn) [inline xml/crm_mon.rng]: When the schema is settled, just ping me, I can detail the data type information
if preferred (perhaps better than the same unstructured information in the comments).

Comment (wenningerk) [reply xml/crm_mon.rng]: Happily accepting your offer ;-)
Ken just merged for master - so go ahead whenever you have time and feel like it.

**Comment by wenningerk** [comment]
synced with master

**Comment by kgaillot** [comment]
CentOS 6 build failing with:
19:12:37 history.c: In function ‘stonith_merge_in_history_list’:
19:12:37 history.c:245: error: void value not ignored as it ought to be

**Comment by kgaillot** [comment]
> Question if it is a good idea to assume code in ASSERT being executed ...

Ah, I see:

    Starting from GLib 2.40, this function returns a boolean value to indicate whether the newly added value was already in the hash table or not.

We only require glib 2.16, so we should lose the assertion in master as well.

**Pull request merged**